internal/dinosql/config.go

```diff
@@ -59,43 +59,43 @@ type Override struct {
 	GoType string `json:"go_type"`
 
 	// fully qualified name of the Go type, e.g. `github.com/segmentio/ksuid.KSUID`
-	PostgresType string `json:"postgres_type"`
+	DbType string `json:"db_type"`
```

Comment:
This change is backwards incompatible and will break existing sqlc users. Configuration changes should be backwards compatible unless we're bumping the configuration version number[0]. There are a few options we have at our disposal.

- Add `db_type`, deprecate `postgres_type`, and complain when both are set. When `postgres_type` is set, we set it's value to `db_type`. 
- Add `mysql_type` and error `postgres_type` and `mysql_type` are both set.

I'm not very happy with either of those options. First, keep your first change, renaming `postgres_type` to `db_type`. However, we'll still allow `postgres_type` to be set as long as `db_type` is unset. 

We will need to add an optional `engine` field to global overrides. This field must be set when two different database engines are in use. Otherwise, we won't know which type to override.

[0] Changing the version number is actually more work because we want to be able to automatically upgrade older versions of the configuration files to new versions.

Reply:
Good thinking with the `engine` field for global overrides. I'll go ahead and make those changes now. Hopefully, these special cases won't make the documentation too dense.